doc_src/en/HowTo_SetUpTeamProject.xml

```diff
@@ -243,6 +244,46 @@
 	</orderedlist>
   </section>
 
+  <section id="how.to.manage.team.project">
+  <title id="how.to.manage.team.project.title">Manage a team project</title>
+
+  <para>Team projects involve a number of management tasks not directly related
+  to translation. Project managers may find it advantageous—or even necessary—to
+  handle such tasks directly from the VCS repository, particularly in cases where
+  the project manager only acts as a coordinator and is not involved in the translation
+  process itself.</para>
+    <variablelist>
+	  <varlistentry>
+	    <term>Deleting files</term>
+	    <listitem>
+		  <para>OmegaT does not provide a mechanism to delete files from a team project.
+		  If any shared team project files have become unnecessary, the project manager
+		  <emphasis>must</emphasis> delete them from the VCS repository.</para>
+	    </listitem>
+	  </varlistentry>
+	  <varlistentry>
+		<term>Adding files</term>
+		<listitem>
+		  <para>Although OmegaT provides a mechanism for adding translation source and
+		  target files to the repository, the project manager must add any other files,
+		  such as dictionaries, glossaries or reference translation memories, from the
+		  VCS repository.</para>
+		</listitem>
+	  </varlistentry>
+	  <varlistentry>
+		<term>Checking project statistics</term>
+		<listitem>
+		  <para>This only applies to workflows that ask translators to commit the translated
+		  target files to the repository as deliverables.</para>
+		  <para>When OmegaT commits the target files, it also commits the project statistics
+		  in <filename>.txt</filename> and <filename>.json</filename> format. This enable the
+		  project manager to consult the statistics, and possibly automate their processing,
+		  without the need to open OmegaT.</para>
```

Comment:
Is that necessary ?

Reply:
Which part? The file types, or the part about the project managers looking at the statistics and automating their processing?

Reply:
The second part. I think burying a possible use at the end of a sentence is the best way to never have it used. Maybe having a small section about automatic processing of OmegaT data in the stats and linking to this section is best.